src/System.Runtime.Extensions/tests/System/MathF.netcoreapp.cs

```diff
@@ -240,6 +240,32 @@ public static void Acos(float value, float expectedResult, float allowedVariance
             AssertEqual(expectedResult, MathF.Acos(value), allowedVariance);
         }
 
+        [Theory]
+        [InlineData( float.NegativeInfinity,  float.NaN,              0.0f)]
+        [InlineData(-0.0f,                    float.NaN,              0.0f)]
+        [InlineData( float.NaN,               float.NaN,              0.0f)]
+        [InlineData( 0.0f,                    float.NaN,              0.0f)]
+        [InlineData( 1.0f,                    0.0f,                   CrossPlatformMachineEpsilon)]
```

Comment:
Similar test here for a value between 0 and 1.  and a negative value that isn't negative zero.